### ISSUE TITLE
api.get_markets() returns: invalid type: null, expected a Decimal type

### ISSUE THREAD

**Issue by peepopoggers** [fabianboesiger/ftx#56]
I have implemented the ftx module inside of my rust application and the command api.get_market(...) works fine.
However, when i use api.get_markets() in the same location, it gives me the following error:

```
Json(Error("invalid type: null, expected a Decimal type representing a fixed-point number", line: 1, column: 17472))
```

Code for reference (inside a module, inside of a switch function):

```rust
        "price"|"p" => {     
            let query = api.get_market(&pair.as_str()).await?;
            println!("  {}: {}", pair, query.price);

            let query2 = api.get_markets().await?;
            println!("{:#?}", query2);
        },
```

In the above, the api.get_market runs fine however the api.get_markets gives me the error as said above. Am I missing something here?

**Comment by MaxFangX** [comment]
Most likely one of the `Market` fields actually returns `null`, which requires us to deserialize as `Option<Decimal>` instead of just `Decimal` in the struct - the API changes frequently and stuff like this happens all the time.

If you uncomment the block commented code in `rest::request` and try `get_markets` again, you should be able to see the API response in a `response.json` file. That can help you figure out what's going on

**Comment by MaxFangX** [comment]
(An easy way to try `get_markets` is to clone this repository if you haven't already, and uncomment the code in `rest::request` in the clone. Then run `cargo test rest::tests::get_markets` after setting up your API key etc)

**Comment by peepopoggers** [comment]
![image](https://user-images.githubusercontent.com/72892531/134912262-2c05a38d-1eab-4585-ab99-05084c489be9.png)
This is an example in ftx docs.

**Comment by MaxFangX** [comment]
I tried it myself, looks like `"last": null,` for `DKNG-1231`:

```
{
  "name": "DKNG/USD",
  "enabled": true,
  "postOnly": false,
  "priceIncrement": 0.005,
  "sizeIncrement": 0.01,
  "minProvideSize": 0.01,
  "last": 51.335,
  "bid": 51.13,
  "ask": 51.465,
  "price": 51.335,
  "type": "spot",
  "baseCurrency": "DKNG",
  "quoteCurrency": "USD",
  "underlying": null,
  "restricted": true,
  "tokenizedEquity": true,
  "highLeverageFeeExempt": true,
  "change1h": 0,
  "change24h": 0,
  "changeBod": 0,
  "quoteVolume24h": 0,
  "volumeUsd24h": 0
},
```

The FTX docs unfortunately don't state whether fields are nullable - we're left to figure it out ourselves

**Comment by MaxFangX** [comment]
I opened a PR for a fix in #57. `get_markets` should work once it's merged in (you can pull the commit onto your own fork in the meantime if you'd like)
